frontend/src/app/libs/event/components/event-description/event-description.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
lets rename lib to `event-description` also route too since in future we will have /event page